powerspikegg/rawdata/tools/fetch/summoner_test.go

```diff
@@ -31,14 +27,18 @@ type mockSummonerFetcherServer struct {
 	response *lolpb.MatchReference
 	// err contains an eventual error to throw back to the client
 	err error
+	// address contains the server address
+	address string
 }
 
-func newMockSummonerFetcherServer(port string) (*mockSummonerFetcherServer, error) {
-	lis, err := net.Listen("tcp", port)
+func newMockSummonerFetcherServer() (*mockSummonerFetcherServer, error) {
+	lis, err := net.Listen("tcp", "127.0.0.1:0")
```

Comment:
You should do this port assignment also on match_test.go for consistency reasons.